packages/vx-drag/src/Drag.js

```diff
@@ -1 +1,98 @@
-export default class Drag {}
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localPoint } from '@vx/event';
+
+export default class Drag extends React.Component {
+  static propTypes = {
+    children: PropTypes.func.isRequired,
+    captureDragArea: PropTypes.bool,
+  };
```

Comment:
might need to add `svg`, `onDragStart ` / `onDragEnd` / `onDragMove`, `width`, `height`, and `captureDragArea` here